src/FuelClient.cc

```diff
@@ -1776,3 +1789,18 @@ void FuelClientPrivate::PopulateLicenses(const ServerConfig &_server)
     ignerr << "Failed to parse license information[" << resp.data << "]\n";
   }
 }
+
+//////////////////////////////////////////////////
+void FuelClientPrivate::CheckForDeprecatedUri(const common::URI &_uri)
+{
+  static std::string oldServer = "fuel.ignitionrobotics.org";
```

Comment:
We should avoid using types with non-trivial destructors in objects of static storage. Can this be replaced with `static const 
char *oldServer = "fuel.ignitionrobotics.org"`?

Reply:
[765bf6c](https://github.com/gazebosim/gz-fuel-tools/pull/294/commits/765bf6c3b3fbb8cb0e996fbe54951883aa9837b4)